src/sentry/seer/similarity/utils.py

```diff
@@ -367,15 +375,28 @@ def has_too_many_contributing_frames(
 
     if contributing_component.frame_counts[key] > MAX_FRAME_COUNT:
         metrics.incr(
-            "grouping.similarity.frame_count_filter",
+            "grouping.similarity.stacktrace_length_filter",
             sample_rate=options.get("seer.similarity.metrics_sample_rate"),
             tags={**shared_tags, "outcome": "block"},
```

Comment:
Should we update this `outcome` to `block_by_frame_count` or something like that?